src/rapidfuzz/process_cpp.py

```diff
@@ -104,17 +104,16 @@ def cpdist(
     import numpy as np
 
     dtype = _dtype_to_type_num(dtype)
-    return np.asarray(
-        _cpdist(
-            queries,
-            choices,
-            scorer=scorer,
-            processor=processor,
-            score_cutoff=score_cutoff,
-            score_hint=score_hint,
-            score_multiplier=score_multiplier,
-            dtype=dtype,
-            workers=workers,
-            **kwargs,
-        )
-    )
\ No newline at end of file
+    distance_matrix = _cpdist(
+        queries,
+        choices,
+        scorer=scorer,
+        processor=processor,
+        score_cutoff=score_cutoff,
+        score_hint=score_hint,
+        score_multiplier=score_multiplier,
+        dtype=dtype,
+        workers=workers,
+        **kwargs,
+    )
+    return np.asarray(distance_matrix).flatten()
```

Comment:
I think we would want to handle this inside the Matrix class already. The problem wit the call to `flatten` is that it leads to a copy of the complete buffer, which I would like to avoid.

Reply:
Got caught up in some other things yesterday. Will make a small PR later today :)

Reply:
No hurry. Just wanted to make sure you don't spend a disproportionate amount of time on this :)